packages/map-template/src/components/Search/Search.jsx

```diff
@@ -265,6 +283,23 @@ function Search({ onSetSize }) {
         }
     });
 
+    useEffect(() => {
+        if (useKeyboard) {
+            const onClick = (event) => {
+                if (event.target.tagName.toUpperCase() === 'DIV' || event.target.tagName.toUpperCase() === 'MI-SEARCH' || event.target.tagName.toUpperCase() === 'INPUT') {
```

Comment:
I think the DIV tagName is a bit too generic. I can trigger the keyboard by clicking on any div element, here the div that acts as container for Mapbox info:

https://github.com/MapsPeople/web-ui/assets/50314165/d322d55c-dfd0-46c7-a786-f11128dcdb00

Reply:
Hmmm, that is true. Is there a way we can target the element's class in this case? We need to target the clicks on the div element with the certain class here 🤔
![image](https://github.com/MapsPeople/web-ui/assets/113503166/2754ab7f-6825-4e02-927b-a8b711913917)

Reply:
Nevermind, I found a way 😄